carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/DriverPool.java

```diff
@@ -368,6 +368,7 @@ private static WebDriver createDriver(String name, DesiredCapabilities capabilit
         }
 
         if (!init) {
+        	//TODO: think about this runtime exception 
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS1135)